### PR TITLE
Upper bound for `docker_hub`

### PR DESCRIPTION
**Pull request by benmandrew**
`docker_hub.0.2.0` has breaking changes, so use a version below that.

### CODE REVIEW

**Comment by benmandrew** [comment]
Other lower-bounds were introduced as well as fixes to satisfy the `(lint-opam)` check.

**Comment by shonfeder** [comment]
Anything blocking this moving forward besides the review from @mtelvers?

**Comment by benmandrew** [comment]
@shonfeder no blockers, but I don't have write access to the repo.

File: opam-health-check.opam [hunk 1]
```diff
@@ -46,8 +46,8 @@ depends: [
   "lwt"
   "lwt_ppx"
   "uri"
-  "docker_hub" {>= "0.1.0"}
-  "memtrace"
+  "docker_hub" {>= "0.1.0" & < "0.2.0"}
```
Comment (shonfeder) [inline opam-health-check.opam:49]: Should we add a TODO noting that we'd like to make updates to use the latest version when feasible? Or is there a reason that we would not plan to do that?

**Review by shonfeder — APPROVED** [review]

File: opam-health-check.opam (hunk 1 above)
Comment (shonfeder) [reply opam-health-check.opam:49]: The new version is a complete API change, and we have no pressing need do incorporate it.

**Pull request merged**